src/fava/util/date.py

```diff
@@ -382,7 +382,7 @@ def parse_date(  # noqa: PLR0911
     match = WEEK_RE.match(string)
     if match:
         year, week = map(int, match.group(1, 2))
-        start = datetime.date.fromisocalendar(year, week + 1, 1)
```

Comment:
this seems to have been my poor attempt at converting from "iso week number" to "%W-like week number".

I think we should switch to using `datetime.datetime.strptime(string, "%Y-W%W").date()`

Reply:
For your info, the tzinfo was suggested by tox, but is useless as Tz is not present in the date we extract afterwards.